src/header/common/content_language.rs

```diff
@@ -0,0 +1,22 @@
+use header::{Language, QualityItem};
+
+header! {
+    #[doc="`Content-Language` header, defined in"]
+    #[doc="[RFC7231](https://tools.ietf.org/html/rfc7231#section-3.1.3.2)"]
+    #[doc=""]
+    #[doc="The `Content-Language` header field describes the natural language(s)"]
+    #[doc="of the intended audience for the representation.  Note that this"]
+    #[doc="might not be equivalent to all the languages used within the"]
+    #[doc="representation."]
+    #[doc=""]
+    #[doc="# ABNF"]
+    #[doc="```plain"]
+    #[doc="Content-Language = 1#language-tag"]
+    #[doc="```"]
```

Comment:
Generating it automatically is not possible with macros currently. But it should be basically a copy & paste job: copy the values from the tests to the docs. Or do you want a program example? (We could create samples for classes of headers)